extensions/amp-analytics/0.1/cookie.js

```diff
@@ -231,3 +231,16 @@ export class CookieWriter {
       });
   }
 }
+
+/**
+ * COOKIE macro resolver
+ * @param {!Window} win
+ * @param {string} name
+ * @return {?string}
+ */
+export function cookieReader(win, name) {
```

Comment:
nit: we could inline this method in variables.js so that we don't need to rename this file. (to avoid the filename conflict with src/cookies.js)

your call.

Reply:
I wanted to keep the methods in cookie.js (formerly cookie-writer.js) because I felt they support the same feature, and we are adding more logic to the `#cookieReader()` method. 

However I realized today I can't do this, because cookie.js and variables.js import from each other then 😢  

I don't want to introduce a `cookie-reader` file, so I'm going to move the method to variables.js. 

However it seems to me variables.js has a lot of unrelated methods. I'd like to move them around later.

Reply:
So I started to move the method to variables.js, and it reminded me there's another reason why I don't want to do that. We perform privacy check on cookie reading/writing. And I really don't like that the `getCookie` `setCookie` function to live everywhere across the code base : (

Any good ideas?